### DISCUSSION TITLE
Food related icons

### DISCUSSION THREAD

**Discussion by usama-365** [tailwindlabs/heroicons#826] [Ideas]
Can we get some food related icons representing common food entitites like burger (not to be confused with hamburger menu icon), pizza, sandwich and beverage etc? It'll really open up countless possibilities for web designers specially in the food website design domain.

**Comment by larshei** [comment]
I agree, I recently needed a coffee mug and a pizza slice and ended up importing the SVGs from fontawesome.

Now, I am looking for a bit of a more generic food icon (like a cutlery set) because on my current project, I would like to indicate wether catering is available or if there are any restaurants nearby.

**Comment by astalarico** [comment]
I would also find this very useful.

**Comment by brianokinyi** [comment]
Was this ever added? I am deeply in need of any food icon
